code/online_challenges/src/codechef/OJUMPS/OJUMPS.c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+
+int main() 
+{
+    long long int a;
+    scanf("%lld", & a);
+    int b = a % 6;
```

Comment:
I think the b should also be of type long long int as in some huge testcases, b can be overflowed.